manifests/client/mount.pp

```diff
@@ -114,4 +117,14 @@
       require  => Nfs::Functions::Mkdir[$mountname]
     }
   }
+
+  if $owner != undef or $group != undef or $mode != undef {
```

Comment:
This will crash when only one or two of the variables are set. I think we should set some default values when there is no value set (maybe in params.pp?).